Xamarin.Forms.Core/BindableObject.cs

```diff
@@ -486,7 +486,7 @@ BindablePropertyContext GetContext(BindableProperty property)
 			for (var i = 0; i < properties.Count; i++)
 			{
 				BindablePropertyContext context = properties[i];
-				if (ReferenceEquals(context.Property, property))
+				if (context != null && ReferenceEquals(context.Property, property))
```

Comment:
I understand this fixes the problem, but I'm more curious to understand how context can be null in this case, and eventually fix that root cause instead of adding a null check.